manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -58,13 +58,8 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
-        - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v1.2.2
-          args:
-            - "--csi-address=$(ADDRESS)"
-          env:
-            - name: ADDRESS
-              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+        - name: csi-snapshot-controller
```

Comment:
snapshot-controller shouldnt be part of driver installation. Should be deployed once per cluster irrespective  of number of csi drivers

Reply:
you are right: 

Both the volume snapshot controller and the CRDs are independent of any CSI driver. Regardless of the number CSI drivers deployed on the cluster, there must be only one instance of the volume snapshot controller running and one set of volume snapshot CRDs installed per cluster.

Reply:
@jichenjc how about, we add CRDs and snapshotcontroller manifests here separately, https://github.com/kubernetes/cloud-provider-openstack/tree/master/manifests and in documentation mention about creating them if not already there in your cluster or we could just link to snapshotcontroller manifests directly from the external-snapshotter repo to avoid syncup issues. But either way , we just need to document about snapshot-controller , and how to deploy it . shouldnt be part of cinder-csi-plugin manifests IMO

Reply:
yes, I addressed in this latest PR